File: model/usuario.py
from utils.db import db
from sqlalchemy import Enum
from dataclasses import dataclass

usuario_types = ('administrador','persona')
usuario_enum = Enum(*usuario_types, name='tipo_usuario')

@dataclass
class Usuario(db.Model):
    __tablename__ = 'usuario'
    id_usuario = db.Column(db.Integer, primary_key=True, autoincrement=True)
    email = db.Column(db.String(100))
    clave = db.Column(db.String(250))
    tipo_usuario = db.Column(usuario_enum, default = 'persona')

    def __init__(self,email,clave, tipo_usuario):
        self.email = email
        self.clave = clave
        self.tipo_usuario = tipo_usuario

File: v2-microservicios/sisvita-app/realizar-test-service/src/main/java/com/realizar_test_service/realizar_test_service/model/model_temporal/Usuario.java
package com.realizar_test_service.realizar_test_service.model.model_temporal;

public class Usuario {

    private Integer idUsuario;
    private String email;
    private String clave;
    private String tipoUsuario;

    public Usuario() {
    }

    public Usuario(Integer idUsuario, String email, String clave, String tipoUsuario) {
        this.idUsuario = idUsuario;
        this.email = email;
        this.clave = clave;
        this.tipoUsuario = tipoUsuario;
    }

    public Integer getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(Integer idUsuario) {
        this.idUsuario = idUsuario;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getClave() {
        return clave;
    }

    public void setClave(String clave) {
        this.clave = clave;
    }

    public String getTipoUsuario() {
        return tipoUsuario;
    }

    public void setTipoUsuario(String tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }
}